test/instrumentation.test.ts

```diff
@@ -108,7 +137,7 @@ describe("bullmq", () => {
   });
 
   afterEach(() => {
-    // printSpans(memoryExporter.getFinishedSpans())
+    // printSpans(memoryExporter.getFinishedSpans());
```

Comment:
```suggestion
```

Reply:
While I understand the want to get rid of commented-out code, I'd like to keep these here for now, out of sheer debugging usefulness.